receiver/kubeletstatsreceiver/README.md

```diff
@@ -37,7 +37,8 @@ endpoint will be used if `auth_type` set to any of the following values:
 - `tls` tells the receiver to use TLS for auth and requires that the fields
 `ca_file`, `key_file`, and `cert_file` also be set.
 - `serviceAccount` tells this receiver to use the default service account token
-to authenticate to the kubelet API.
+to authenticate to the kubelet API along with the default certificate which is signed by the cluster's root CA cert.
+Those can be found at the (receiver's implementation)[https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.94.0/internal/kubelet/client.go#L25-L26].
```

Comment:
Instead of linking to the constants lets put the string right in the readme, that seems a little friendlier.

Reply:
Updated: [b8166bb](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/31280/commits/b8166bb4dedcef46e22c69f8a865f19f68878ea4)